src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -46,7 +47,12 @@ public static Query buildSearchConceptsQuery(
     var domainClause =
         createDomainClause(conceptTablePointer, conceptTableVariable, domainOption.getName());
 
-    List<SelectExpression> select = List.of(nameField, idField, countField);
+    List<SelectExpression> select =
+        List.of(
+            nameField,
+            idField,
+            countField,
+            new SelectAlias(new Literal(1), HierarchyQueryBuilder.HAS_CHILDREN));
```

Comment:
This will produce the SQL `SELECT <...>, 1 AS has_children`

Reply:
Can you include this in the Java doc comment? I think line 38 would now be:
`sql_code = c.concept_name, c_concept_id, COUNT(DISTINCT co.person_id) AS count, 1 AS has_children FROM`